lib/recipients/project.js

```diff
@@ -39,9 +39,12 @@ module.exports = async ({ schema, logger, task }) => {
 
   const raDate = project.raDate && moment(project.raDate).format(dateFormat);
   const revocationDate = project.revocationDate && moment(project.revocationDate).format(dateFormat);
-  const ropsDate = project.revocationDate && moment(project.revocationDate).add(28, 'days').format(dateFormat);
+  const endDate = project.revocationData || project.expiryDate;
```

Comment:
```suggestion
  const endDate = project.revocationDate || project.expiryDate;
```

Reply:
Well that's a mess isn't it.

I'll a test that deal with that case too.